modules/linuxboot

```diff
@@ -21,7 +27,7 @@ linuxboot_configure := \
 	if [ "$(linuxboot_board)" = "qemu" ]; then \
 		echo >&2 "Pre-building edk2 OVMF" ; \
 		( cd $(build)/$(linuxboot_base_dir)/edk2/OvmfPkg ; \
-		./build.sh -n $$CPUS \
+		./build.sh -n $(CPUS) \
```

Comment:
@tlaurion What do you say to making this change a separate PR and merging it in? I was about to make one fixing this before seeing this change in this PR.

Reply:
@synackd please do. Same should be made under coreboot module

Reply:
In hindsight, I probably could have combined #940 and #941 into one PR, though it does categorically separate the linuxboot and coreboot modules...

Reply:
Have you tested it? The idea here is to pass CPUS to command line
make BOARD= x230 CPUS=2

There is no such invocation as CPUS as a command. 
@synackd?

Reply:
These changes can be retracted since they are already merged as of #940.